src/services/NotificationService.ts

```diff
@@ -24,3 +24,41 @@ export const sendCredentialService = async ({
     return error;
   }
 };
+
+
+export const readUserId = async (
+  userId?: string | string[],
+  fieldValue?: boolean
+): Promise<any> => {
+  let apiUrl: string = `${process.env.NEXT_PUBLIC_MIDDLEWARE_URL}/user/v1/read/${userId}?fieldvalue=false`;
```

Comment:
@shreyas1434shinde check if this is present in other service?

Reply:
@itsvick comment resolved